recipes/kmodes/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "kmodes" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2c3f86028e5bff5213a5ac77544c79a1eb59389de59c410a22f5ac0501e85bec
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+  run:
+    - numpy >=1.10.4
+    - scikit-learn 0.19.*
+    - scipy >=0.13.3
+
+test:
+  imports:
+    - "{{ name }}"
```

Comment:
Can you add a few more import tests? eg `kmodes.kprototypes` Also please hard code the module.